cliboa/template/requirements.above36.txt

```diff
@@ -49,6 +49,7 @@ pandas==0.25.3
 paramiko==2.8.1
 proto-plus==1.19.8; python_version >= '3.6'
 protobuf==3.19.1; python_version >= '3.6'
+psycopg2-binary==2.9.1
```

Comment:
not reflect it in requirements.txt
pip install will fail.
so I added it here as well.